src/monster.cpp

```diff
@@ -825,7 +825,7 @@ void monster::melee_attack(Creature &target, bool, matec_id) {
             if (u_see_me) {
                 //~ $1s is monster name, %2$s is that monster target name,
                 //~ $3s is target bodypart name in accusative, 4$s is target armor name.
-                add_msg(_("The %1$s hits %2$s %3$s but is stopped by %2$s %4$s."), name().c_str(),
```

Comment:
This works, but you could also insert a redundant positional parameter.
"The %1$s hits %2$s %3$s but is stopped by %4$s %5$s.", name().c_str(), target.disp_name(true).c_str(), body_part_name_accusative(bp_hit).c_str(), target.disp_name(true).c_str(), target.skin_name().c_str());